src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/CommitmentOrchestrator.cs

```diff
@@ -761,10 +761,11 @@ private CommitmentListItemViewModel MapFrom(CommitmentView listItem, string late
             };
         }
 
-        private async Task<List<ITrainingProgramme>> GetTrainingProgrammes()
+        private async Task<List<ITrainingProgramme>> GetTrainingProgrammes(bool includeFrameworks)
```

Comment:
As discussed, this method is duplicated across EmployerCommitments & ProviderApprenticeshipService, so we could push common code down into a common assembly. It would however be a pain point to generate the nuget package, and we couldn't use a shared project as the solutions are in separate repos.

Perhaps we could think of a nice solution to this?